include/rosdiscover-clang/Ast/Function.h

```diff
@@ -163,10 +166,15 @@ class SymbolicFunctionCall : public virtual SymbolicStmt {
     for (auto const &entry : args) {
       argsJson[entry.first] = entry.second->toJson();
     }
+    auto j_deps = nlohmann::json::array();
```

Comment:
Super minor nitpick: suggest renaming to `jDeps` or similar to maintain camelcase convention rather than switching to snakecase

Reply:
Fixed